src/lib/ndpi_main.c

```diff
@@ -9547,19 +9547,56 @@ int ndpi_get_category_id(struct ndpi_detection_module_struct *ndpi_str, char *ca
 
 /* ****************************************************** */
 
+
+char *ndpi_default_ports_string(char *ports_str,u_int16_t *default_ports){
```

Comment:
static and remove `ndpi_` prefix, please

Reply:
Done. For possible future reference what is the principle to have a ndpi function prefix ?